flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -0,0 +1,360 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.flink</groupId>
+		<artifactId>flink-connectors</artifactId>
+		<version>1.9-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<artifactId>flink-connector-hive_${scala.binary.version}</artifactId>
```

Comment:
Why it depends on scala version ?

Reply:
There's no actual code dependency on scala. Flink needs to built against a specific scala version for scala APIs (https://ci.apache.org/projects/flink/flink-docs-stable/flinkDev/building.html#scala-versions). Other connectors also do it this way.

Reply:
I know some flink modules depends on scala, but for these connectors, it is very confusing to add scala version in artifactId. If user develop a flink app using java, it would confuse him to add flink-connector-hive_2.11 as dependency. Besides that, it would lead multiple duplicated release artifacts due to different scala version. But actually these artifacts are the same.

Reply:
Although flink need to specify scala version to build it, it doesn't mean every module must specify scala version and has scala version in its artifactId.

Reply:
I think only the modules depends on a scala module (e.g. `flink-streaming-java_2.11`) need to specify the scala version. Other modules shouldn't specify scala version. For example, `flink-core`, `flink-table-common`, `flink-table-api-java` do not have a scala version.

Reply:
I get it. However, I would suggest to 
(1). include `flink-streaming-java` in this PR or 
(2) include `flink-streaming-java` when needed with setup scala suffix